extensions/common/json-ld/src/test/java/org/eclipse/edc/jsonld/transformer/to/TestInput.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.jsonld.transformer.to;
+
+import com.apicatalog.jsonld.document.JsonDocument;
+import jakarta.json.JsonObject;
+
+import static com.apicatalog.jsonld.JsonLd.expand;
+
+/**
+ * Functions for shaping test input.
+ */
+public class TestInput {
+
+    /**
+     * Expands test input as Json-ld is required to be in this form
+     */
+    public static JsonObject getExpanded(JsonObject message) {
+        try {
+            return expand(JsonDocument.of(message)).get().asJsonArray().getJsonObject(0);
```

Comment:
why don't use our `TitaniumJsonLd` service instead?

Reply:
It's a lot less verbose, and we would have to inject a mock monitor. I figured we only need to do a simple expansion, and we use a similar method in other modules that don't have a direct dependency on the Json-ld module. We could look into switching this, but it would result in a lot more test code.